de1plus/profile.tcl

```diff
@@ -520,41 +520,110 @@ namespace eval ::profile {
         return $profile_type
     }
         
-    # Based on proc load_settings_vars, but only loads profile variables, and returns a list with the profile (which can be
-    # coerced to an array) instead of loading the profile into the global settings. The source file can also be a shot 
-    # file, and only the profile-related vars will be loaded from it, plus any variables given in list $extra_vars.
-    proc read_legacy { profile_path {extra_vars {}} } {
-        if { [file pathtype $profile_path] eq "relative" && [file dirname $profile_path] eq "." } {
-            set profile_path "[homedir]/profiles/[file tail $profile_path]"
+    proc profile_type_text { profile_type {beverage_type {}} } {
+        switch [fix_profile_type $profile_type] \
+        settings_2a {
+            set type "Pressure"
+        } settings_2b {
+            set type "Flow"
+        } settings_2c {
+            set type "Advanced"
+        } default {
+            set type "Unknown"
         }
-        if { [file extension $profile_path] eq "" } {
-            append profile_path ".tcl"
+        
+        switch [lindex $beverage_type 0] \
```

Comment:
dont we maybe want to strip the string here first?

Reply:
Thing is I found some profiles having `beverage_type {pourover}` instead of `beverage_type pourover`, so I wanted to handle those.

Reply:
meant something like:
switch  [ string trim [lindex $beverage_type 0] " " ]

Reply:
But I think the parsing of the profile file should prevent `beverage_type` to contain spaces.

Reply:
Doesn't hurt, in any case. Changed.